<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>exileSniffer</class>
 <widget class="QMainWindow" name="exileSniffer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>827</width>
    <height>673</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>exileSniffer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="processTabs">
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="interceptionTab">
       <attribute name="title">
        <string>Interception</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QFrame" name="statusFrame">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>100</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QFormLayout" name="formLayout">
           <item row="3" column="1">
            <widget class="QLabel" name="statusInfoText">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Waiting for processes named &quot;PathOfExile.exe&quot; or &quot;PathOfExile_x64.exe&quot;</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
             <property name="indent">
              <number>12</number>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="statusInfoLabel">
             <property name="font">
              <font>
               <pointsize>13</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Action:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="statusLabel">
             <property name="font">
              <font>
               <pointsize>13</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Status:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLabel" name="statusText">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>13</pointsize>
              </font>
             </property>
             <property name="text">
              <string>No running Path of Exile clients found</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
             <property name="indent">
              <number>12</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="decodeTab">
       <attribute name="title">
        <string>Decoder</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="rawDecryptTab">
       <attribute name="title">
        <string>Raw Plaintext</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0">
        <property name="spacing">
         <number>2</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="textPaneFrame" stretch="3,1,0">
          <item>
           <widget class="QTextEdit" name="ptHexPane">
            <property name="font">
             <font>
              <family>Courier New</family>
              <pointsize>8</pointsize>
             </font>
            </property>
            <property name="lineWrapMode">
             <enum>QTextEdit::NoWrap</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="ptASCIIPane"/>
          </item>
          <item>
           <widget class="QScrollBar" name="verticalScrollBar">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="rawControls">
          <item>
           <widget class="QFrame" name="bytesPerRowFrame">
            <property name="maximumSize">
             <size>
              <width>200</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="bytesRowLael">
               <property name="maximumSize">
                <size>
                 <width>86</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>Bytes Per Row:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="bytesRowCombo">
               <property name="maximumSize">
                <size>
                 <width>60</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="editable">
                <bool>true</bool>
               </property>
               <item>
                <property name="text">
                 <string>16</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>32</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>48</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>64</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="rawLinewrapCheck">
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Line Wrap</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame">
            <property name="maximumSize">
             <size>
              <width>259</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="filterLabel">
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>0 Packets Filtered</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="filtersBtn">
               <property name="maximumSize">
                <size>
                 <width>84</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>Filters</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="metaLogTab">
       <attribute name="title">
        <string>Log</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QPlainTextEdit" name="metaLog"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>827</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuGame_Instances">
    <property name="title">
     <string>Game Processes</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
   </widget>
   <addaction name="menuGame_Instances"/>
   <addaction name="menuSettings"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="exileSniffer.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>filtersBtn</sender>
   <signal>clicked()</signal>
   <receiver>exileSniffer</receiver>
   <slot>showRawFiltersDLG()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>628</x>
     <y>608</y>
    </hint>
    <hint type="destinationlabel">
     <x>413</x>
     <y>336</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bytesRowCombo</sender>
   <signal>activated(QString)</signal>
   <receiver>exileSniffer</receiver>
   <slot>rawBytesRowChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>277</x>
     <y>607</y>
    </hint>
    <hint type="destinationlabel">
     <x>413</x>
     <y>336</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rawLinewrapCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>exileSniffer</receiver>
   <slot>toggleRawLineWrap(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>382</x>
     <y>608</y>
    </hint>
    <hint type="destinationlabel">
     <x>413</x>
     <y>336</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>showRawFiltersDLG()</slot>
  <slot>rawBytesRowChanged(QString)</slot>
  <slot>toggleRawLineWrap(bool)</slot>
 </slots>
</ui>
